src/main/java/bio/terra/tanagra/query/bigquery/BigQueryExecutor.java

```diff
@@ -24,39 +31,36 @@ public BigQueryExecutor(GoogleBigQuery bigQuery) {
 
   @Override
   public QueryResult execute(QueryRequest queryRequest) {
-    String sql = queryRequest.getSql();
+    String sql = renderSQL(queryRequest.query());
     LOGGER.info("Running SQL against BigQuery: {}", sql);
     TableResult tableResult = bigQuery.queryBigQuery(sql);
 
     Iterable<RowResult> rowResults =
         Iterables.transform(
             tableResult.getValues(),
             (FieldValueList fieldValueList) ->
-                new BigQueryRowResult(fieldValueList, queryRequest.getColumnHeaderSchema()));
+                new BigQueryRowResult(fieldValueList, queryRequest.columnHeaderSchema()));
 
-    return new QueryResult(rowResults, queryRequest.getColumnHeaderSchema());
+    return new QueryResult(rowResults, queryRequest.columnHeaderSchema());
   }
 
   @Override
-  public String executeAndExportResultsToGcs(QueryRequest queryRequest, String gcsBucketName) {
-    // TODO: Add study and dataset names.
-    String fileName =
-        "tanagra_export_dataset_"
-            + System.currentTimeMillis()
-            // GCS file name must be in wildcard format.
-            // https://cloud.google.com/bigquery/docs/reference/standard-sql/other-statements#export_option_list:~:text=The%20uri%20option%20must%20be%20a%20single%2Dwildcard%20URI
-            + "_*.csv";
-
-    String sql =
-        String.format(
-            "EXPORT DATA OPTIONS(uri='gs://%s/%s',format='CSV',overwrite=true,header=true) AS %n%s",
-            gcsBucketName, fileName, queryRequest.getSql());
-    LOGGER.info("Running SQL against BigQuery: {}", sql);
-    bigQuery.queryBigQuery(sql);
+  public void createTableFromQuery(TableId destinationTable, String sql, boolean isDryRun) {
+    throw new NotImplementedException();
```

Comment:
Are these not implemented because we don't care about BQ?

Reply:
This is an API that no longer works and any code calling it needs to be ported, so I added this to remind myself to fix it later. Once code that uses it has been rewritten the API can be removed.

The reason why it no longer works is because it depends on creating a new table from a query of an existing table, and that can only work if the existing (source) table is in the same system as the index table, which isn't the case in azure. There may still be some valid cases if the code has a source mapping of index -> index, which is possible in tanagra.